lib/Mojo/Pg.pm

```diff
@@ -46,7 +46,7 @@ sub from_string {
   return $self unless $str;
   my $url = Mojo::URL->new($str);
   croak qq{Invalid PostgreSQL connection string "$str"}
-    unless $url->protocol eq 'postgresql';
+    unless $url->protocol =~ m{^(?:postgresql|postgres)$};
```

Comment:
Quoting with `m{...}` there seems like a pretty arbitrary choice, and doesn't reflect current best practices.